Makefile

```diff
@@ -70,26 +79,27 @@ dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
+BASE_CONFIG := ./desktop-config/config-base.json
+TARGET_CONFIG := ./desktop-config/config-$(CONFIG_ENV).json
 
-BASE_CONFIG := $(THIS_DIR)/desktop-config/config-base.json
-ENV_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
-
-.PHONY: desktop$/config.json
-desktop$/config.json:
-ifeq (,$(wildcard $(ENV_CONFIG)))
+.PHONY: desktop/config.json
```

Comment:
same here - did you experience problems with `$/`?

Reply:
Yep - this in fact didn't work on Windows (see comment above).